libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -266,5 +266,15 @@ public async Task ProcessAsync(HttpRequest httpRequest, HttpResponse httpRespons
                 }
             }
         }
+
+        /// <summary>
+        /// Factory method to create the <see cref="FacebookMessage"/> instance of the <see cref="Activity"/> to be sent to Facebook.
+        /// </summary>
```

Comment:
I might add a note here saying something like: "This allows an override to add a Facebook-defined message tag to a message sent to Facebook."